recipes/zip-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: zip-cos6-x86_64
+  version: 3.0
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/zip-3.0-1.el6_7.1.x86_64.rpm
+    sha256: 64624b07c11db01d0f74a578a9dd93d20cfc2be4d317190d21d8c71c00adecf5
+    no_hoist: true
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/zip-3.0-1.el6_7.1.src.rpm
+    folder: source
+    no_hoist: true
+
+build:
+  number: 2
+  noarch: generic
+  missing_dso_whitelist:
+    - '*'
+
+test:
+  commands:
+    - test -d "${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot"
+
+about:
+  home: http://www.info-zip.org/Zip.html
+  license: BSD
+  # license_file: in the rpm source
+  license_family: BSD
+  summary: (CDT) A file compression and packaging utility compatible with PKZIP
+  description: |
+    The zip program is a compression and file packaging utility.  Zip is analogous
+    to a combination of the UNIX tar and compress commands and is compatible with
+    PKZIP (a compression and file packaging utility for MS-DOS systems).  Install
+    the zip package if you need to compress files using the zip program.
+
+extra:
+  recipe-maintainers:
+    - conda-forge/Core
```

Comment:
Please add yourself and anybody else interested.

Reply:
Why not core?

Reply:
Nobody is interested but this has to be done. Thus I think this falls to core

Reply:
I don't want to be a maintainer of this useless feedstock. So, please leave me out of it.

Reply:
Got it. Please send a list of cdts you think are useful so that we can avoid this.